kibbeh/public/locales/hi/translation.json

```diff
@@ -241,12 +241,12 @@
       "modal": {
         "needsFuture": "भविष्य में होना चाहिए",
         "roomName": "रूम का नाम",
-        "minLength": "काम से काम 2",
+        "minLength": "कम से कम 2",
         "roomDescription": "रूम के बारे में"
       }
     },
     "roomChat": {
-      "title": "चाट",
+      "title": "गपशप",
```

Comment:
lol changing chat to "गपशप" isn't a nice idea since it means chit chat/gossip.

Reply:
sure will revert